packages/providers/src.ts/base-provider.ts

```diff
@@ -751,44 +750,70 @@ export class BaseProvider extends Provider {
      *
      * @param txId - id of the transaction to search for
      */
-    async getTransaction(txId: string | Promise<string>): Promise<TransactionResponse> {
+    async getTransaction(transactionId: string | Promise<string>): Promise<TransactionResponse> {
+        //currently considers only previewnet!
         await this.getNetwork();
```

Comment:
I don't think we need the `getNetwork` call. I think I've removed it in `develop`

Reply:
Removed the `getNetwork` call